server/grpc_service.go

```diff
@@ -239,6 +239,8 @@ func (s *Server) StoreHeartbeat(ctx context.Context, request *pdpb.StoreHeartbea
 
 const regionHeartbeatSendTimeout = 5 * time.Second
 
+var heartbeatStreamRebindInterval = time.Minute
```

Comment:
can we use a config here to speed up test? 

I see that you should wait at least 1m in test to rebind.

Reply:
I've set it to 3 seconds in test.

Reply:
oh, this is a var I just notice. Seem it is a not good way here.